src/Plane.js

```diff
@@ -0,0 +1,5 @@
+class Plane {
+
+}
+
```

Comment:
could've added some variables and methods here and use it for more tests, while calling it from my airport class